nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -296,15 +292,15 @@ public boolean isContractAnnotation(String annotationName) {
     return contractAnnotations.contains(annotationName);
   }
 
-  protected Set<MethodClassAndName> getKnownInitializers(Set<String> qualifiedNames) {
+  protected ImmutableSet<MethodClassAndName> getKnownInitializers(Set<String> qualifiedNames) {
```

Comment:
Note this is called only once during config loading in `ErrorProneCLIFlagsConfig`